script/ghci-flags

```diff
@@ -26,11 +27,11 @@ function flags {
 
   # preprocessor options, for -XCPP
   echo "-optP-include"
-  echo "-optP$build_products_dir/autogen/cabal_macros.h"
+  echo "-optP$build_dir/semantic-0.10.0.0/noopt/build/autogen/cabal_macros.h"
 
   # autogenerated sources, both .hs and .h (e.g. Foo_paths.hs)
-  echo "-i$build_products_dir/autogen"
-  echo "-I$build_products_dir/autogen"
+  echo "-i$build_dir/semantic-0.10.0.0/noopt/build/autogen"
+  echo "-I$build_dir/semantic-0.10.0.0/noopt/build/autogen"
```

Comment:
Nice; my bad for breaking this in the first place.

I am mildly surprised that these are ending up in `noopt`, as this doesn’t seem to be the case with other projects I’ve been working with. Is that a result of a `optimization: false` setting in a `cabal.project.local` file by any chance? (I’m mostly concerned with how to ensure that this is portable to everyone’s config.)

Reply:
@robrix I do have that set, yes. I’ve unset it and a testing.

Reply:
It occurs to me that one way to ensure portability might involve including redundant paths both with `noopt` and without, but that might also result in `ghc` not being sure which module to load, which could be a problem.